game-physics/game-physics-eng.svg

```diff
@@ -0,0 +1,1734 @@
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:11.25px;line-height:1.25999999;font-family:Oxygen;-inkscape-font-specification:Oxygen;fill:#800000"
+         sodipodi:role="line"
+         x="122.69712"
+         y="768.80359"
+         id="tspan7717">d</tspan></text>
+    <text
+       id="text7721"
+       y="768.89764"
+       x="212.59843"
+       style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;line-height:0%;font-family:'Oxygen Mono';-inkscape-font-specification:'Oxygen Mono';text-align:start;letter-spacing:0px;word-spacing:0px;text-anchor:start;fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
+       xml:space="preserve"><tspan
+         y="768.89764"
+         x="212.59843"
+         sodipodi:role="line"
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:11.25px;line-height:1.25999999;font-family:Oxygen;-inkscape-font-specification:Oxygen"
+         id="tspan7725">Circles are intersecting if: <tspan
+   style="fill:#800000"
+   id="tspan7737">d</tspan><tspan
+   style="font-size:64.99999762%;baseline-shift:super"
+   id="tspan7735">2</tspan> ≤ <tspan
+   style="fill:#008000"
+   id="tspan7739">r</tspan><tspan
+   style="font-size:64.99999762%;baseline-shift:sub;fill:#008000"
+   id="tspan7733">1</tspan><tspan
+   style="font-size:64.99999762%;baseline-shift:super"
+   id="tspan7727">2</tspan> + <tspan
+   style="fill:#000080"
+   id="tspan7741">r</tspan><tspan
+   style="font-size:64.99999762%;baseline-shift:sub;fill:#000080"
+   id="tspan7731">2</tspan><tspan
+   style="font-size:64.99999762%;baseline-shift:super"
+   id="tspan7729">2</tspan></tspan></text>
+    <text
+       id="text7745"
+       y="566.92908"
+       x="24.80315"
+       style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;line-height:0%;font-family:'Oxygen Mono';-inkscape-font-specification:'Oxygen Mono';text-align:start;letter-spacing:0px;word-spacing:0px;text-anchor:start;fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
+       xml:space="preserve"><tspan
+         y="566.92908"
+         x="24.80315"
+         sodipodi:role="line"
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:11.25px;line-height:1.25999999;font-family:Oxygen;-inkscape-font-specification:Oxygen"
+         id="tspan7749">Turtle moves a distance d in direction α.</tspan><tspan
+         y="581.10406"
+         x="24.80315"
+         sodipodi:role="line"
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:11.25px;line-height:1.25999999;font-family:Oxygen;-inkscape-font-specification:Oxygen"
+         id="tspan632">How to convert it into x, y coordinates (and back?</tspan><tspan
```

Comment:
Missing closing parenthesis: `How to convert it into x, y coordinates (and back?`